test-it/karma/javascriptinvoke/DataTypes.Test.js

```diff
@@ -66,6 +66,46 @@ describe('A JavaScript function invoke', function () {
         window.NI_StringFunction = function (value) {
             return value;
         };
+
+        window.NI_Int8ArrayFunction = function (value) {
+            expect(value instanceof Int8Array).toBeTruthy();
```

Comment:
prefer toBeTrue

Truthiness is definitely a flaw of JS, should avoid relying on implicit type conversions